safaribooks.py

```diff
@@ -275,6 +275,8 @@ def __init__(self, args):
 
         self.cookies = {}
 
+        self.session = None
+
```

Comment:
Create a session variable to store session after login and use it for all following requests